app/src/main/java/com/example/tasky/authentication/di/AuthenticationModule.kt

```diff
@@ -15,6 +21,10 @@ import retrofit2.Retrofit
 @InstallIn(SingletonComponent::class)
 class AuthenticationModule {
 
+    @Provides
+    fun providesSharedPreferences(@ApplicationContext context: Context): SharedPreferences =
+        context.getSharedPreferences("tasky_cache", MODE_PRIVATE)
```

Comment:
You are storing a user's token
Thats a private information. You wouldnt want someone to get that info from the SharedPreferences (if they have a Rooted device)


You should create an EnceyptedSharedPrwference, to store sensitive info